trieste/acquisition/function.py

```diff
@@ -790,11 +795,15 @@ def __init__(
 
         if penalizer is None:
             penalizer = soft_local_penalizer
-
         self._lipschitz_penalizer = penalizer
 
+        if base_acquisition_function_builder is None:
+            base_acquisition_function_builder = ExpectedImprovement()
+        self._base_builder = base_acquisition_function_builder
```

Comment:
You could put these all on one line as follows: (and may be able to avoid having to give a type annotation this way too)

```suggestion
        self._base_builder = ExpectedImprovement() if base_acquisition_function_builder is None else base_acquisition_function_builder
```